src/ProductConstructionService/ProductConstructionService.Api/Program.cs

```diff
@@ -37,12 +45,34 @@
 
 builder.AddWorkitemQueues(credential);
 
+if (!builder.Environment.IsDevelopment())
+{
+    builder.AddAuthentication();
+
+}
+
 builder.AddServiceDefaults();
 
 // Add services to the container.
 
 builder.Services.AddControllers();
-builder.Services.AddSwaggerGen();
+builder.Services.AddSwaggerGen(options =>
+{
+    options.AddSecurityDefinition(
+                    "Bearer",
+                    new OpenApiSecurityScheme
+                    {
+                        Type = SecuritySchemeType.ApiKey,
+                        In = ParameterLocation.Header,
+                        Scheme = "bearer",
+                        Name = HeaderNames.Authorization
+                    });
+
+    options.AddSecurityRequirement(new OpenApiSecurityRequirement
+                {
+                    {new OpenApiSecurityScheme{Reference = new OpenApiReference{Id = "Bearer", Type = ReferenceType.SecurityScheme}}, Array.Empty<string>()},
```

Comment:
What's up with the indent/formatting in this section?

Reply:
my bad, didn't see it when I copied it